src/content/en/updates/2017/06/supercharged-codesplit.md

```diff
@@ -0,0 +1,279 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We implemented code splitting and route based chunking just like WebPack.
+
+{# wf_updated_on: 2017-06-22 #}
+{# wf_published_on: 2017-06-22 #}
+{# wf_tags: supercharged #}
```

Comment:
Adding new tags is fine when they'll be used consistently and more than once. Please add the tag to `src/data/_commonTags.json` so that it doesn't yell at you.